src/compas_fab/robots/configuration.py

```diff
@@ -240,3 +247,111 @@ def scaled(self, scale_factor):
         config = self.copy()
         config.scale(scale_factor)
         return config
+
+    def differences_generator(self, other):
```

Comment:
I guess `iter_differences()` would be a better name here, however, I wonder why exactly this is an iterator and not a list? Is it an extremely heavy operation that needs to be taken step by step? Or is it only for the convenience of using `yield` in the middle of the method?

Reply:
well it is not so heavy, but it depends on the number of configuration values. But for example, if you want to compare 2 configurations for the RFL (34 values?) and you only want to know if they are the same (see `close_to` method), then you want to stop the loop as soon as they are not.
I'll change the naming.